src/feedback/models/base.py

```diff
@@ -205,13 +204,6 @@ class AlternativVorname(models.Model):
     vorname = models.CharField(_('first name'), max_length=30, blank=True)
     person = models.ForeignKey(Person, on_delete=models.CASCADE)
 
-    @staticmethod
```

Comment:
the function should be there. While merging there were two copies of the function where I removed the copy. Besides I checked the file in GitHub and it looked ok, but maybe you could take a look too, if I am missing something.